addons/website_livechat/static/src/components/discuss/discuss.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
-    <t t-inherit="mail.Discuss.content" t-inherit-mode="extension">
-        <xpath expr="//*[hasclass('o_Discuss_thread')]" position="before">
-            <t t-if="discuss.thread.visitor">
+    <t t-inherit="mail.ThreadView" t-inherit-mode="extension">
+        <xpath expr="//*[hasclass('o_ThreadView_topbar')]" position="after">
```

Comment:
Hello, there is still one comment from @alexkuhn that was not done / probably not clear enough:

The current code still links the visitor banner feature to the topbar feature, and we would like to remove this link.

The override to introduce the banner at the correct position should be on the `t-if` of the topbar, not the topbar itself.

And then, here there should be a condition to clearly states when the banner should be shown: if there is a visitor + repeat the condition that was on topbar (-> must be in discuss)

Reply:
Note: if the condition becomes too complex, introduce a boolean compute field to clearly define it.